### PR TITLE
Initial state of a Control value

### PR DESCRIPTION
**Pull request by bubriks**
[#38](https://github.com/Wappsto/wappsto-python/issues/38)

### CODE REVIEW

**Comment by bubriks** [comment]
not done most likely needs more work

File: test/wappsto_test.py [hunk 1]
```diff
@@ -53,15 +53,15 @@ def create_response(self, verb, callback_exists, trace_id, bulk):
     
     if verb == "DELETE" or verb == "PUT" or verb == "GET":
         if trace_id is not None:
-            trace = '"meta": {"trace": "'+str(trace_id)+'"},'
+            trace = {"trace": str(trace_id)}
 
-        message = '{"jsonrpc": "2.0", "id": "1", "params": {"url": "'+url+'",'+trace+' "data": {"meta": {"id": "'+id+'"}, "data": "93"}}, "method": "'+verb+'"}'
+        message = {"jsonrpc": "2.0", "id": "1", "params": {"url": str(url), "meta": trace, "data": {"meta": {"id": id}, "data": 44}}, "method": verb}
```
Comment (AndreasBomholtz) [inline test/wappsto_test.py]: Data should be a string

Comment (AndreasBomholtz) [reply test/wappsto_test.py]: eg 44 => "44"

**Comment by bubriks** [comment]
Add Deprecated for last_controlled needs to be done before the merge

**Pull request merged**